CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
> So you don't think a CoC is harmful, just unnecessary? Yeah, that's pretty much where I stand on this too.

A purely agenda-driven CoC could very easily be harmful in theory. However, the one as currently written has been balanced by feedback from the Opal maintainers and open source community. I don't see much room for harm from it (or maybe I'm not malicious/creative enough?).